src/pages/iou/IOUModal.js

```diff
@@ -54,8 +56,19 @@ class IOUModal extends Component {
      *
      * @returns {String}
      */
+
     getTitleForStep() {
-        return steps[this.state.currentStepIndex] || '';
+        let title = '';
+        const amount = this.state.iouAmount;
+
+        if (this.state.currentStepIndex === 1) {
+            title = this.props.hasMultipleParticipants ? 'Spilt' : 'Request';
```

Comment:
Also, it's a bit weird how you're reassigning `title` in here. I think this is much cleaner:

```javascript
getTitleForStep() {
    if (this.state.currentStepIndex === 1) {
        return `${this.props.hasMultipleParticipants ? 'Split' : 'Request'} $${this.state.iouAmount}`;
    }
    
    return steps[this.state.currentStepIndex] || '';
}
```